frontend/src/features/admin/agencies/AgencyFilterBar.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { useKeycloak } from '@react-keycloak/web';
+import { render } from '@testing-library/react';
+import { IAgencyFilter } from 'interfaces';
+import TestCommonWrapper from 'utils/TestCommonWrapper';
+
+import { AgencyFilterBar } from './AgencyFilterBar';
+
+const mockValues: IAgencyFilter = {};
+const mockAdd = jest.fn();
+const mockChange = jest.fn();
+
+jest.mock('@react-keycloak/web');
+
+(useKeycloak as jest.Mock).mockReturnValue({
+  keycloak: {
+    userInfo: {
+      agencies: [1],
+      roles: [],
+    },
+    subject: 'test',
+  },
+});
+
+it('renders correctly...', () => {
```

Comment:
As agencies are nota  big part of this application I am leaving the test at a snapshot for now in case of removal